docs/user-guide/running.md

```diff
@@ -15,6 +15,55 @@ When your environment is running, you can [deploy your workload](deploying-workl
 
 To force device re-registration remove EdgeDevice CR from the cluster and `/var/local/yggdrasil/device` directory from the device
 
+---
+## Automatically
+
+### Prerequisites
+- Clone all Flotta components: [flotta-operator](https://github.com/project-flotta/flotta-operator), [flotta-device-worker](https://github.com/project-flotta/flotta-device-worker) and [yggdrasil](https://github.com/RedHatInsights/yggdrasil).
+- On minikube: add to `/etc/hosts`: `<minikube ip> <host name>`.
+
+### Start Flotta operator
+Script arguments: 
+- Image registry and tag (for example: `IMG=quay.io/project-flotta/flotta-operator:latest`) 
+- Target (k8s or ocp) 
+- Path to operator directory (optional)
+
+From the operator directory run:
+```
+bash ./hack/start-operator.sh IMG=<image> TARGET=<target>
+```
+Note: if running outside the operator directory, pass also `OPERATOR_PATH=<operator-path>`.
+
+### Start Flotta agent
+Script arguments: 
+- Target (k8s or ocp)
```

Comment:
I think that actually expected argument names need to be mentioned here.

Reply:
Done